docs/docs/vulnerability/languages/java.md

```diff
@@ -0,0 +1,51 @@
+# Java
+
+## Features
+Trivy supports three types of Java scanning: `jar/war/par/ear`, `pom.xml` and `*gradle.lockfile` files.
+The following table provides an outline of the features Trivy offers.
+
+
+| Artifact         |  Maven repository [^1]   | Dev dependencies |
+|------------------|:------------------------:|:-----------------|
+| jar/war/par/ear  |            ✓             | Include          |
+| pom.xml          |            ✓             | Exclude          |
+| *gradle.lockfile |            -             | Exclude          |
+
+### Jar
+To find information about your jar[^2] file, Trivy parses `pom.properties` and `MANIFEST.MF` files in your jar[^2] file and takes `required properties`[^3].
+
+    If those files don't exist or don't contain enough information - Trivy uses [maven repository REST API](https://central.sonatype.org/search/rest-api-guide) to try to get the missing information.
+
+!!! Note
+    Trivy only takes information about packages. We don't take a list of vulnerabilities for packages from the `maven repository`(Information about data sources for java you can see [here](../detection/data-source.md))
```

Comment:
it looks like a space is missing between `maven repository` and (Information :)

Reply:
actually there are a few other places where a space is missing in front of an opening parenthesis in the file :)

Reply:
Thanks!
Fixed.